must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -107,7 +113,11 @@ for ns in $namespaces; do
     if [ "${operatorImage}" = "" ]; then
         echo "not able to find the rook's operator image. Skipping collection of ceph command output" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
     elif [[ $cephClusterCount -gt 0 ]]; then
-        apply_helper_pod "$ns" "$operatorImage"
+        if [[ $(oc get csv -n "${ns}" --no-headers | awk '{print $5}') -gt 4.5.0 ]]; then
```

Comment:
What about 4.5.z (e.g. 4.5.1/4.5.2)? Will this condition cover that too?

Reply:
no, updated